test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ContextTests.cs

```diff
@@ -0,0 +1,59 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Core.Features;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.E2E.Common;
+using Microsoft.Health.Fhir.Tests.E2E.Rest;
+using Microsoft.Health.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Shared.Tests.E2E.Rest
+{
+    [Trait(Traits.Category, Categories.Context)]
+    [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]
+    public class ContextTests : IClassFixture<HttpIntegrationTestFixture>
```

Comment:
>ContextTests [](start = 17, length = 12)

nit: Maybe rename this to `FhirRequestContextTests` or `HttpContextTests` to give more context :)

Reply:
[https://www.hl7.org/fhir/http.html#custom](https://www.hl7.org/fhir/http.html#custom) since they called "Custom headers" I'll call it CustomHeadersTests :)

---
In reply to: [590550963](https://github.com/microsoft/fhir-server/pull/1728#discussion_r590550963) [](ancestors = 590550963)